monitoring/alerts.rb

```diff
@@ -0,0 +1,216 @@
+require "json"
+require "google/cloud/monitoring"
+require "google/cloud/monitoring/v3/alert_policy_service_client"
```

Comment:
move requires into the relevant region tags

Reply:
moved to respective examples.